src/main.rs

```diff
@@ -61,6 +63,33 @@ fn load_filetypes_database(matches: &ArgMatches, user_config_path: &PathBuf) ->
     }
 }
 
+fn available_theme_names(user_config_path: &PathBuf) -> Result<Vec<String>> {
```

Comment:
Shouldn't we list the user-themes in addition to the builtin themes?

Reply:
Updated, this should hopefully work now. It captures all possible theme paths. 

I ran a copy to test out.

```
> cp themes/ayu.yml $HOME/.config/vivid/themes/ayu2.yml
> ./target/debug/vivid themes
ayu
ayu2
jellybeans
molokai
snazzy
solarized-dark
solarized-light